### PR TITLE
[PW-2831] Configure Bullet in development mode

### PR DESCRIPTION
**Pull request by prasanthkrajan**
# Refer to the issue id if any. Include the link to the issue. E.g:
[Configure Bullet in development mode](https://perxtechnologies.atlassian.net/browse/PW-2831)

# Describe the changes made. What does this PR changes that might be critical. If any critical decisions have been made, make sure you explain the rationale for these decisions.

- made this
- did that

# How does the implementation addresses the problem

After merging this, what are we providing extra.

### CODE REVIEW

File: lib/core/config/settings.yml [hunk 1]
```diff
@@ -94,3 +94,7 @@ smtp:
   ssl: false
   tls: false
   from: 'no-reply@example.com'
+
+bullet:
```
Comment (rpbaltazar) [inline lib/core/config/settings.yml]: As per our discussion, this is not needed and should be replaced with the settings in the config/deployment as i showed you

**Comment by rpbaltazar** [comment]
I saw your PR in https://github.com/PerxTech/whistler-services/pull/290/files
The default configuration should still live in the Ros OSS because people wouldn't have access to our platform settings

**Comment by prasanthkrajan** [comment]
> I saw your PR in https://github.com/PerxTech/whistler-services/pull/290/files
> The default configuration should still live in the Ros OSS because people wouldn't have access to our platform settings

Updated in ROS

File: lib/core/ros-core.gemspec [hunk 2]
```diff
@@ -39,6 +39,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'trailblazer-activity-dsl-linear', '~> 0.2.1'
   spec.add_dependency 'warden', '1.2.8'
   spec.add_dependency 'zero-rails_openapi', '2.1.0'
+  spec.add_dependency 'bullet'
```
Comment (rpbaltazar) [inline lib/core/ros-core.gemspec]: This should be a developmnent dependency not an actual code dependency

**Pull request merged**